source/extensions/quic_listeners/quiche/active_quic_listener_config.cc

```diff
@@ -0,0 +1,27 @@
+#include "extensions/quic_listeners/quiche/active_quic_listener_config.h"
+
+#include "envoy/api/v2/listener/quic_config.pb.h"
+
+#include "server/well_known_names.h"
+
+#include "extensions/quic_listeners/quiche/active_quic_listener.h"
+
+namespace Envoy {
+namespace Quic {
+
+ProtobufTypes::MessagePtr ActiveQuicListenerConfigFactory::createEmptyConfigProto() {
+  return std::make_unique<envoy::api::v2::listener::QuicProtocolOptions>();
+}
+
+Network::ActiveUdpListenerFactoryPtr
+ActiveQuicListenerConfigFactory::createActiveUdpListenerFactory(const Protobuf::Message& message) {
+  auto& config = dynamic_cast<const envoy::api::v2::listener::QuicProtocolOptions&>(message);
```

Comment:
Can you use downcastAndValidate here?

Reply:
There is no validation annotation in quic_config.proto